pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -72,6 +73,9 @@ func (p *ConfigurationEditFlags) markFlagMakesRevision(f string) {
 func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 	command.Flags().StringVar(&p.Image, "image", "", "Image to run.")
 	p.markFlagMakesRevision("image")
+	command.Flags().StringVar(&p.ImagePullPolicy, "imagePullPolicy", "",
```

Comment:
I wonder if we should just call it `pullPolicy` instead to save people typing? While I like consistency with the yaml, we've already deviated on other things, like `--concurrency-limit` instead of `--container-concurrency`, and I think dropping `image` is less controversial.   (yes, I'd love to switch to just `--concurrency` since the use of the word `limit` is less intuitive to me, but that's a different topic)

Reply:
also - should it be `--pull-policy` since we seem to use `-`s instead of camelCase ?

Reply:
Yes, option should be dash separated. Also ok with use pull policy (but wouldn't mind image pull policy, too).

Reply:
done